notebooks/spark/AerospikeSparkScala.ipynb

```diff
@@ -521,7 +1282,8 @@
     "aerospike.recordspersecond : same as java client\n",
     "\n",
     "#### Other\n",
-    "aerospike.keyType : Primary key type hint for schema inference. Always set it properly if primary key type is not string\n",
+    "- aerospike.keyType : Primary key type hint for schema inference. Always set it properly if primary key type is not string\n",
+    "- aerospike.digestType : Digest type hint for schema inference. Always set it properly if digest type is not byte[]\n",
     "\n",
     "See https://www.aerospike.com/docs/connect/processing/spark/reference.html for detailed description of the above properties"
```

Comment:
Remove this section because it's covered better on the config page. Let's not maintain the same info in two places. Please point the users to the config page right in the beginning just like the Python notebook.